### PR TITLE
Create creating-interactive-prototypes.md

### PR DESCRIPTION
**Pull request by akemalakcay**

### CODE REVIEW

File: checklists/creating-interactive-prototypes.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Creating Interactive Prototypes
+category: Design
+date: "2020-03-11"
+tags: ['tools', 'prototyping', 'design-process']
+description: Creating interactive prototypes is an important step on the design process. Use this checklist to make sure you cover the basics of creating them!
```
Comment (ecempehlivan) [inline checklists/creating-interactive-prototypes.md]:
```suggestion
description: Creating interactive prototypes is an important step in the product itself. Use this checklist to make sure you cover the basics of creating them!
```

**Comment by akemalakcay** [comment]
> It would be better if you mention technical sides like; "If something sticky then do it sticky on the prototype too" or "if something is lightbox then it should appear with lightbox pop in effect" and etc.. @akemalakcay

Good thinking! I think those are more-of a reminders rather than to-do's, don't you agree? @alpcanaydin

**Comment by alpcanaydin** [comment]
> > It would be better if you mention technical sides like; "If something sticky then do it sticky on the prototype too" or "if something is lightbox then it should appear with lightbox pop in effect" and etc.. @akemalakcay
> 
> Good thinking! I think those are more-of a reminders rather than to-do's, don't you agree? @alpcanaydin

Absolutely! Then you can add more to-do's for them 👊

File: checklists/creating-interactive-prototypes.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+- [ ] Create the screens  
+These screens don't have to be high-fidelity designs.
+
+- [ ] Wire them up  
+Make sure you have linked every screen to each other so that ther are no gaps in interactions.
+
+- [ ] Pay attention to component behaviours  
+For better prototyping you should pay extra attention to component states and their behaviours. E.g. sticky components, hover and active effects, micro-interactions...
```
Comment (alpcanaydin) [inline checklists/creating-interactive-prototypes.md]: You can be more explicit here. I think you can divide the examples to separated to-do's

Comment (akemalakcay) [reply checklists/creating-interactive-prototypes.md]: Reason why I am not specifying those into this checklist is that I am planning on to create another checklist that covers the essentials of the prototyping processes. This checklist here is more general than the one I will create soon. 

Is that a bad idea? @alpcanaydin

**Pull request merged**